lib/simple_segment/client.rb

```diff
@@ -70,6 +74,7 @@ def group(options)
     # @option :context [Hash]
     # @option :integrations [Hash]
     # @option :timestamp [#iso8601] (Time.now)
+    # @option :message_id
```

Comment:
It feels weird adding this here, because [the docs](https://segment.com/docs/connections/sources/catalog/libraries/server/ruby/#alias) say that `Alias` only supports `userId` and `previousId`. But I just followed suite with what's already here.

Reply:
I think that is OK 👍